src/status_im2/contexts/wallet/home/view.cljs

```diff
@@ -59,5 +45,11 @@
                         {:render-fn               quo/token-value
                          :data                    temp/tokens
                          :content-container-style {:padding-horizontal 8}}]
-         :collectibles [collectibles]
-         [activity])])))
+         :collectibles [quo/empty-state
+                        {:title        (i18n/label :t/no-collectibles)
+                         :description  (i18n/label :t/no-collectibles-description)
+                         :placeholder? true}]
+         [quo/empty-state
+          {:title        (i18n/label :t/no-activity)
+           :description  (i18n/label :t/empty-tab-description)
+           :placeholder? true}])])))
```

Comment:
Not for this PR: We need to move these to `status-im2.contexts.wallet.home.tabs.view` ns.